discovery/file/file.go

```diff
@@ -39,18 +39,23 @@ import (
 )
 
 var (
+	fileSDReadErrorsCount = prometheus.NewCounter(
+		prometheus.CounterOpts{
+			Name: "prometheus_sd_file_read_errors_total",
+			Help: "The number of File-SD read errors.",
+		})
 	fileSDScanDuration = prometheus.NewSummary(
 		prometheus.SummaryOpts{
 			Name:       "prometheus_sd_file_scan_duration_seconds",
 			Help:       "The duration of the File-SD scan in seconds.",
 			Objectives: map[float64]float64{0.5: 0.05, 0.9: 0.01, 0.99: 0.001},
 		})
-	fileSDReadErrorsCount = prometheus.NewCounter(
+	fileSDTimeStamp        = NewTimestampCollector()
+	fileWatcherErrorsCount = prometheus.NewCounter(
 		prometheus.CounterOpts{
-			Name: "prometheus_sd_file_read_errors_total",
-			Help: "The number of File-SD read errors.",
+			Name: "prometheus_inotify_errors_total",
```

Comment:
```suggestion
			Name: "prometheus_sd_file_inotify_errors_total",
```

Reply:
👍 should have noticed 😅 Thank you for that feedback.

Fixed up in [305b754](https://github.com/prometheus/prometheus/pull/11066/commits/305b7544431651ff0af5c5b8c474be6db6ed7121) - as well as updated the help text to decouple from `inotify`-specific language.